src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEAssemblySymbol.cs

```diff
@@ -233,20 +228,11 @@ public override bool MightContainExtensionMethods
         {
             get
             {
-                if (!_lazyContainsExtensionMethods.HasValue())
-                {
-                    var moduleSymbol = this.PrimaryModule;
-                    var module = moduleSymbol.Module;
-                    // The F# compiler may not emit an assembly-level ExtensionAttribute, and previous versions of C# never checked for it.
-                    // In order to avoid a breaking change (while preserving the perceived performance benefits of not looking for extension
-                    // methods in assemblies that don't contain them), we'll also look for FSharpInterfaceDataVersionAttribute.
-                    var mightContainExtensionMethods = module.HasExtensionAttribute(_assembly.Handle, ignoreCase: false) ||
-                                                       module.HasFSharpInterfaceDataVersionAttribute(_assembly.Handle);
```

Comment:
Should we also remove these helper methods (at least the "HasFSharpInterfaceDataVersionAttribute" one...assuming it's not used elsewhere)?